tools/dmm_scanner/scan_camera_ctags.py

```diff
@@ -1,48 +0,0 @@
-import os
```

Comment:
Don't delete this file, it is a test to ensure the cameras tags are unique, and should remain

Reply:
replied in another comment